pipelines/pom.xml

```diff
@@ -54,6 +54,11 @@
     <auto-value.version>1.10.4</auto-value.version>
   </properties>
 
+  <!-- The artifact versions defined in the dependencyManagement section below are the versions that
```

Comment:
I think this comment block is common Maven knowledge, isn't it? If yes, then maybe we should remove it? If you want to keep something, maybe replace it just with a link to [the official documentation](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html).

Reply:
Yes, this is common maven knowledge. I have removed the comments.